README.md

```diff
@@ -69,7 +69,7 @@ See [Wrappers](WRAPPERS.md)
 
 ## Release and branch model
 
-We release every month on the last Tuesday. These release are minor releases, i.e. 2.0.0 to 2.1.0. In between there might be bugfix/hotfix/patch releases when needed, i.e. 2.0.1.
+We release every month on the frist Tuesday. These release are minor releases, i.e. 2.0.0 to 2.1.0. In between there might be bugfix/hotfix/patch releases when needed, i.e. 2.0.1.
```

Comment:
There is a typo if **first**

Reply:
And another one in the next sentence. It should be _These release**s**_

Reply:
fixed